### ISSUE TITLE
module 'nncf' has no attribute 'Dataset'

### ISSUE THREAD

**Issue by DiKu-nn** [openvinotoolkit/nncf#1538]
I have install nncf, but i can not use nncf.Dataset

**Comment by vshampor** [comment]
Greetings!

Please describe your problem more specifically.

**Comment by DiKu-nn** [comment]
#====================================================================
NNCF provides a suite of advanced algorithms for Neural Networks inference optimization in OpenVINO with minimal accuracy drop. We will use 8-bit quantization in post-training mode (without the fine-tuning pipeline) to optimize YOLOv7.

Note: NNCF Post-training Quantization is available as a preview feature in OpenVINO 2022.3 release.

Fully functional support will be provided in the next releases.

The optimization process contains the following steps:

1.Create Dataset for quantization
2.Run nncf.quantize for getting optimized model
3.Serialize OpenVINO IR model using openvino.runtime.serialize function

Reuse validation dataloader in accuracy testing for quantization. For that, it should be wrapped into the nncf.Dataset object and define transformation function for getting only input tensors.

#=================================================================================

**i cannot use nncf.quantize and nncf.Dataset**

**Comment by chengven027-intellif** [comment]
I met the same error today when I use `pip install nncf`.
And I try this command `pip install -r requirements.txt, python setup.py install`, then it works.

**Comment by alexsu52** [comment]
Hello folks, `nncf.quantize` and `nncf.Dataset` will be introduced in NNCF 2.4.0. Also you can use NNCF from the release branch https://github.com/openvinotoolkit/nncf/tree/release_v240.

NNCF version you can check as following code:
```
import nncf
print(nncf.__version__)
```

**Comment by DiKu-nn** [comment]
> 

ok, thanks for your reply
